breathecode/marketing/receivers.py

```diff
@@ -36,3 +40,18 @@ def cohort_post_save(sender, instance, created, *args, **kwargs):
         ac_academy = ActiveCampaignAcademy.objects.filter(academy__id=instance.academy.id).first()
         if ac_academy is not None:
             add_cohort_slug_as_acp_tag.delay(instance.id, instance.academy.id)
+
+
+@receiver(event_saved, sender=Event)
+def event_save(sender, instance, *args, **kwargs):
+    ac_academy = ActiveCampaignAcademy.objects.filter(academy__id=instance.academy.id).first()
+    if ac_academy is not None:
```

Comment:
Should be better it just occurs if the event is being created like `downloadable_post_save`